climakitae/util/generate_gwl_tables.py

```diff
@@ -151,22 +156,35 @@ def get_table_one_cesm2(
             smoothed.to_array(dim="scenario", name=model).dropna("time").to_pandas()
         )
         gwlevels = pd.DataFrame()
-        for level in [1.5, 2, 3, 4]:
-            gwlevels[level] = get_gwl(oneModel.T, level)
-        return gwlevels
+        for level in list(np.linspace(1.5, 4.00, 251)):
+            gwlevels[round(float(level), 3)] = get_gwl(oneModel.T, level)
+
+        # Modifying and returning oneModel to be seen as a WL lookup table with timestamp as index, to get the average WL across all simulations.
+        final_model = oneModel.T
+        final_model.columns = ens_mem + "_" + final_model.columns
+        return gwlevels, final_model
 
     def get_table_cesm2(
```

Comment:
I haven't been following the development of this as much as I should, but I am wondering why we are putting specific model code in here? Utils are usually generalized not specific to one data set.